docs/benchmarks.html

```diff
@@ -231,113 +231,128 @@ <h1>Benchmarks</h1>
                     canvas.className = 'benchmark-chart';
                     graphDetails.appendChild(canvas);
 
-                    // Create chart
-                    const data = {
-                        datasets: [{
-                            label: name,
-                            data: dataset.map(d => {
-                                return { x: new Date(d.commit.timestamp), y: d.bench.value };
-                            }),
-                            borderColor: '#000000',
-                            backgroundColor: '#000000' + '60'
-                        }]
-                    };
-                    const options = {
-                        scales: {
-                            xAxes: [{
-                                type: 'time',
-                                time: {
-                                    unit: 'day',
-                                    displayFormats: {
-                                        day: 'M/D'
+                    // Store chart data for lazy creation
+                    let chartInstance = null;
+
+                    function createChart() {
+                        if (chartInstance) return; // Already created
+
+                        const data = {
+                            datasets: [{
+                                label: name,
+                                data: dataset.map(d => {
+                                    return { x: new Date(d.commit.timestamp), y: d.bench.value };
+                                }),
+                                borderColor: '#000000',
+                                backgroundColor: '#000000' + '60',
+                                lineTension: 0
```

Comment:
This disables chart smoothing (leads to some odd looking shapes with spiky data).